src/_css/main.css

```diff
@@ -113,7 +113,7 @@ body {
 }
 #header h1 a {
   padding: 1em;
-  font-size: 16px;
+  font-size: 1em;
```

Comment:
Turns out this should've been `rem`. Whoops! https://github.com/v8/v8.dev/commit/a25843d09a6b0f32a74171d5edd8dc0199a91bf1